setup.py

```diff
@@ -14,7 +14,7 @@
     version=package["version"],
     author=package['author'],
     author_email=package['author-email'],
-    packages=[package_name],
+    packages=[package_name,package_name+".utils"],
```

Comment:
Could you go over this change? how would that affect PyPI?

Reply:
I reverted it to the old one. I am not sure why added +".utils" there seems to have been an import issue in the past but somehow it is now working without the +".utils".